Framework/Kernel/src/TimeSeriesProperty.cpp

```diff
@@ -509,8 +509,11 @@ void TimeSeriesProperty<TYPE>::splitByTimeVector(const std::vector<DateAndTime>
 
   // check inputs
   if (timeToFilterTo.size() != inputWorkspaceIndicies.size() + 1) {
-    throw std::runtime_error("Input time vector's size does not match(one more larger than) target "
-                             "workspace index vector's size inputWorkspaceIndicies.size() \n");
+    // throw std::runtime_error("Input time vector's size does not match(one more larger than) target "
```

Comment:
Remove commented out code

Reply:
@bolotovskyr Git blame shows it's you who commented this code. Can you say why? I couldn't find any use of this method anywhere except the unit tests so I have no clue how and when it should be used. Lets me to wonder if we should remove this method!

Reply:
@jmborr That was an oversight.  I commented it out while debugging a unit test and forgot to uncomment back it after I fixed the test.

Reply:
What about `TimeSeriesPropertyTest.h` where `splitByTimeVector()` is used? Are we removing it?

Reply:
we have to check this a bit more. I don't understand in which context we're actually splitting a `TimeSeriesProperty `because the whole point of the `TimROI `is to stop splitting the `TimeSeriesProperty  `objects 🤔

Reply:
Please add comments about that to [EWM742](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20(Change%20Management)#action=com.ibm.team.workitem.viewWorkItem&id=742). It should be a separate PR from this